CONTRIBUTING.md

```diff
@@ -113,6 +115,22 @@ install it and then run with:
 cargo deny --all-features check all
 ```
 
+## Other implementations
+
+The Coreutils has different implementations, with different levels of completions:
+
+* [GNU's](https://git.savannah.gnu.org/gitweb/?p=coreutils.git)
+* [BSD](https://github.com/openbsd/src/tree/master/bin)
+* [Busybox](https://github.com/mirror/busybox/tree/master/coreutils)
+* [Toybox (Android)](https://github.com/landley/toybox/tree/master/toys/posix)
```

Comment:
Is toybox really only android? I thought it was basically the same idea as busybox?

Reply:
https://landley.net/toybox/about.html
`Toybox's main goal is to make Android [self-hosting](http://landley.net/aboriginal/about.html#selfhost) by improving Android's command line utilities so it can build an installable Android Open Source Project image entirely from source under a stock Android system.`

Reply:
Alright, I stand corrected :)